tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -206,6 +208,7 @@ def class_host_ft(self,
         skip_yum_update_during_provisioning(template='Kickstart default finish')
         host = entities.Host(
             architecture=module_architecture,
+            build=True,
```

Comment:
Above requires to get the correct `Build` status during assert .. it was missed in the last PR.